docs/src/capi/howto/external-etcd.md

```diff
@@ -0,0 +1,117 @@
+# CAPI External Etcd
+
+To replace the built-in [dqlite][dqlite] database with an external etcd to
+manage the kubernetes state in the CAPI workload cluster follow this
+`how-to guide`.
+This example shows how to create a 3-node workload cluster with an external
+etcd. 
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- Clusterctl
+- A CAPI management cluster
+- Initialize ClusterAPI
+- Installed CRDs
+- Secured 3-node etcd deployment
+
+Please refer to the [getting-started][getting-started] for instructions.
+
+## Create Kubernetes Secrets
+
+Create three kubernetes secrets:
+
+- {cluster-name}-etcd-servers
+- {cluster-name}-etcd
+- {cluster-name}-apiserver-etcd-client
+
+```{note}
+Replace {cluster-name} with the name of your cluster.
+It is important to follow this naming convention since the providers will be looking for these names.
+Please note that this example uses `c1` for the cluster name.
+```
+
+Create the secret for the etcd servers:
+
+```
+kubectl apply -f - <<EOF 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: c1-etcd-servers
+  namespace: default
+
+data:
+  servers: $(echo -n "https://etcd-1:2379,https://etcd-2:2379,https://etcd-3:2379" | base64 -w0)
+EOF
+```
+
+Please export the path to your etcd certs directory:
+
+```
+export CERTS_DIR=path/to/etcd-certs
+```
+
+Create the secret for the etcd root ca:
+
+```
+kubectl create secret generic c1-etcd \
+  --from-file=ca.crt="$CERTS_DIR/etcd-root-ca.pem"
+```
+
+Create the `c1-apiserver-etcd-client` secret:
+
+```
+kubectl create secret tls c1-apiserver-etcd-client \
+  --cert=$CERTS_DIR/etcd-1.pem --key=$EXT_ETCD_DIR/etcd-1-key.pem 
```

Comment:
Where does $EXT_ETCD_DIR come from? Did I miss it?

Reply:
Good catch, yes I forgot to commit the update to the name.